lib/bolt/project.rb

```diff
@@ -8,8 +8,8 @@ class Project
     BOLTDIR_NAME = 'Boltdir'
     PROJECT_SETTINGS = {
       "name"  => "The name of the project",
-      "plans" => "An array of plan names to whitelist. Whitelisted plans are included in `bolt plan show` output",
-      "tasks" => "An array of task names to whitelist. Whitelisted plans are included in `bolt task show` output"
+      "plans" => "An array of plan names to show. These plans are included in `bolt plan show` output",
+      "tasks" => "An array of task names to show. These tasks are included in `bolt task show` output"
```

Comment:
I wonder if the plans and tasks keys should really be `displayed_plans` and `displayed_tasks` or something similar.  

From the key name you might think you're supposed to put all of your tasks/plans in that key.  Granted the description should help clear up that ambiguity.

Reply:
Hmm this loses a subtle distinction which is that these aren't the names _to_ show, they're the names that _can_ be shown if they exist. If you don't have a task with that name, it won't show up even if it's in this list. Maybe "An array of task names that are part of the project. These tasks are included in `bolt task show` output"?

Reply:
"An array of plan names to show. These plans are included in `bolt plan show` output as long as they also exist inside the project"

or 

"An array of plan names to show, if they exist in the project. These plans are included in `bolt plan show` output"